src/Symfony/Bundle/WebServerBundle/WebServer.php

```diff
@@ -145,12 +145,16 @@ public function isRunning($pidFile = null)
      */
     private function createServerProcess(WebServerConfig $config)
     {
-        $finder = new PhpExecutableFinder();
-        if (false === $binary = $finder->find()) {
-            throw new \RuntimeException('Unable to find the PHP binary.');
+        $executable = $config->getExecutable();
+
+        if ($executable === null) {
+            $finder = new PhpExecutableFinder();
+            if (false === $executable = $finder->find()) {
+                throw new \RuntimeException('Unable to find the PHP executable.');
+            }
```

Comment:
I'd move this logic to WebServerConfig constructor now that it holds an executable.